Makefile

```diff
@@ -1,9 +1,21 @@
 .PHONY: test
-test:
+
+test: export CQ_DEST_PG_TEST_CONN=postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
```

Comment:
lets keep the test case target as it is as the cq one. We can add a new target for custom test cases added by us. To avoid any conflicts in this file.